packages/cq-cert-runner/README.md

```diff
@@ -0,0 +1,45 @@
+# `cq-cert-runner`
+
+CQ Certification Runner by Metriport Inc.
```

Comment:
ditto: CQ > Carequality

Likely not needed to replace in all places, but as with other acronyms, we introduce it in the beginning of the text and can reference it later.